src/main/java/me/maxouxax/multi4j/crous/RestoCrous.java

```diff
@@ -0,0 +1,32 @@
+package me.maxouxax.multi4j.crous;
+
+public class RestoCrous {
+
+    String NomRu;
+    String ImageRu;
+    String DateRu;
+    String MenuRu;
```

Comment:
Please follow Java variables naming convention.

```suggestion
    String nomRu;
    String imageRu;
    String dateRu;
    String menuRu;
```